OptimizelySDK/Data Model/Audience/ConditionHolder.swift

```diff
@@ -95,9 +95,6 @@ extension Array where Element == ConditionHolder {
     }
     
     func evaluate(op: LogicalOp, project: ProjectProtocol?, attributes: OptimizelyAttributes?) throws -> Bool {
-        guard self.count > 0 else {
```

Comment:
why is this removed?

Reply:
This condition is covered by the caller, so the guard-else condition never happens